logging/telemetryConfig.go

```diff
@@ -33,6 +33,9 @@ import (
 // TelemetryConfigFilename default file name for telemetry config "logging.config"
 var TelemetryConfigFilename = "logging.config"
 
+var DefaultTelemetryUsername = "telemetry-v9"
+var DefaultTelemetryPassword = "oq%$FA1TOJ!yYeMEcJ7D688eEOE#MGCu"
```

Comment:
use lower-case for `DefaultTelemetryUsername` . i.e. `defaultTelemetryUsername`
and the same for DefaultTelemetryPassword, since we don't want to export these.